README.md

```diff
@@ -1,84 +1,31 @@
 <!--
 
-This source file is part of the Stanford Biodesign Digital Health Next.js Template open-source project
+This source file is part of the Stanford Biodesign Digital Health Spezi Web Configurations open-source project
 
-SPDX-FileCopyrightText: 2023 Stanford University and the project authors (see CONTRIBUTORS.md)
+SPDX-FileCopyrightText: 2024 Stanford University and the project authors (see CONTRIBUTORS.md)
 
 SPDX-License-Identifier: MIT
 
 -->
 
-# Biodesign Digital Health Next.js Template
+# Biodesign Digital Health Spezi Web Configurations
 
-[![Build and Test](https://github.com/StanfordBDHG/NextJSTemplate/actions/workflows/build-and-test.yml/badge.svg)](https://github.com/StanfordBDHG/NextJSTemplate/actions/workflows/build-and-test.yml)
-[![Deployment](https://github.com/StanfordBDHG/NextJSTemplate/actions/workflows/main.yml/badge.svg)](https://github.com/StanfordBDHG/NextJSTemplate/actions/workflows/main.yml)
-[![codecov](https://codecov.io/gh/StanfordBDHG/NextJSTemplate/graph/badge.svg?token=dfQW5eZ2up)](https://codecov.io/gh/StanfordBDHG/NextJSTemplate)
+[![Build and Test](https://github.com/StanfordSpezi/spezi-web-configurations/actions/workflows/build-and-test.yml/badge.svg)](https://github.com/StanfordSpezi/spezi-web-configurations/actions/workflows/build-and-test.yml)
+[![Deployment](https://github.com/StanfordSpezi/spezi-web-configurations/actions/workflows/main.yml/badge.svg)](https://github.com/StanfordSpezi/spezi-web-configurations/actions/workflows/main.yml)
+[![codecov](https://codecov.io/gh/StanfordSpezi/spezi-web-configurations/graph/badge.svg?token=dfQW5eZ2up)](https://codecov.io/gh/StanfordSpezi/spezi-web-configurations)
 [![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.10052055.svg)](https://doi.org/10.5281/zenodo.10052055)
 
-## How To Use This Template
```

Comment:
Would be great if we can add a few paragraphs of content & explanation of this repo and how one can use the resulting package & setup for their own project in here to make this more beginner-friendly & ensure that we start having a good overview and documentation in our web-related repos.